### PR TITLE
AUI-2822

### PR DESCRIPTION
**Pull request by blzaugg**

### CODE REVIEW

**Comment by jonmak08** [comment]
Reviewed and Merged.

See https://github.com/liferay/alloy-ui/tree/1.5.2.33.

File: build/aui-form-validator/aui-form-validator-debug.js [hunk 1]
```diff
@@ -620,6 +620,7 @@ var FormValidator = A.Component.create({
 		},
 
 		_defErrorFieldFn: function(event) {
+			console.log('_defErrorFieldFn()');
```
Comment (blzaugg) [inline build/aui-form-validator/aui-form-validator-debug.js]: Sorry @jonmak08, I missed this.

File: build/aui-form-validator/aui-form-validator.js [hunk 2]
```diff
@@ -620,6 +620,7 @@ var FormValidator = A.Component.create({
 		},
 
 		_defErrorFieldFn: function(event) {
+			console.log('_defErrorFieldFn()');
```
Comment (blzaugg) [inline build/aui-form-validator/aui-form-validator.js]: Sorry @jonmak08, I missed this.

File: build/aui-form-validator/aui-form-validator-debug.js (hunk 1 above)
Comment (jonmak08) [inline build/aui-form-validator/aui-form-validator-debug.js]: No worries.  I caught it.

File: build/aui-form-validator/aui-form-validator.js (hunk 2 above)
Comment (jonmak08) [inline build/aui-form-validator/aui-form-validator.js]: No worries. I caught it.

**Pull request closed without merging**